src/libMediator/Mediator.h

```diff
@@ -90,6 +90,9 @@ class Mediator {
   std::shared_ptr<std::deque<std::pair<PubKey, Peer>>> m_DSCommittee;
   std::mutex m_mutexDSCommittee;
 
+  std::shared_ptr<std::deque<std::pair<PubKey, Peer>>> m_initialDSCommittee;
```

Comment:
Can we define a alias as using DequeOfDSCommittee = std::deque<std::pair<PubKey, Peer>>?